apps/opik-backend/src/main/java/com/comet/opik/domain/FeedbackScoreDAO.java

```diff
@@ -381,18 +433,56 @@ public Mono<List<String>> getFeedbackScoreNames(UUID projectId, boolean withExpe
         });
     }
 
-    private void bindStatementParam(UUID projectId, Statement statement) {
+    private static Mono<List<String>> getNames(Statement statement) {
+        return makeMonoContextAware(bindWorkspaceIdToMono(statement))
+                .flatMapMany(result -> result.map((row, rowMetadata) -> row.get("name", String.class)))
+                .distinct()
+                .collect(Collectors.toList());
+    }
```

Comment:
I always prefer private methods to come after the public ones. Can you move it to be below the public methods?